connect-example/src/main/java/com/stripe/android/connect/example/di/ServiceModule.kt

```diff
@@ -0,0 +1,40 @@
+package com.stripe.android.connect.example.di
+
+import android.content.Context
+import com.stripe.android.connect.example.data.EmbeddedComponentManagerWrapper
+import com.stripe.android.connect.example.data.EmbeddedComponentService
+import com.stripe.android.connect.example.data.SettingsService
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object ServiceModule {
```

Comment:
I don't think this is necessary if we use the `@Singleton` annotation appropriately, right?

Reply:
Huh, I didn't realize this annotation existed. Much better, and yes we shouldn't need this module anymore!